src/test/java/lambda/part1/exercise/Exercise1.java

```diff
@@ -0,0 +1,137 @@
+import lambda.data.Person;
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.List;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+
+public class Exercise1 {
+
+    @Test
+    public void sortPersonsByAgeUsingArraysSortComparator() {
+        Person[] persons = getPersons();
+
+        // TODO использовать Arrays.sort
+        class compareByAge implements Comparator<Person>{
+            @Override
+            public int compare(Person person1, Person person2){
+                return Integer.compare(person1.getAge(), person2.getAge());
+            }
+        }
+
+        Arrays.sort(persons, new compareByAge());
+
+        assertArrayEquals(new Person[]{
+                new Person("Иван", "Мельников", 20),
+                new Person("Николай", "Зимов", 30),
+                new Person("Алексей", "Доренко", 40),
+                new Person("Артем", "Зимов", 45)
+        }, persons);
+
+        assertArrayEquals(new Person[]{
+                new Person("Иван", "Мельников", 20),
+                new Person("Николай", "Зимов", 30),
+                new Person("Алексей", "Доренко", 40),
+                new Person("Артем", "Зимов", 45)
+        }, persons);
+    }
+
+    @Test
+    public void sortPersonsByAgeUsingArraysSortAnonymousComparator() {
+        Person[] persons = getPersons();
+
+        // TODO использовать Arrays.sort
+        Arrays.sort(persons, new Comparator<Person>() {
+            @Override
+            public int compare(Person person1, Person person2) {
+                return Integer.compare(person1.getAge(), person2.getAge());
+            }
+        });
+
+        assertArrayEquals(new Person[]{
+                new Person("Иван", "Мельников", 20),
+                new Person("Николай", "Зимов", 30),
+                new Person("Алексей", "Доренко", 40),
+                new Person("Артем", "Зимов", 45)
+        }, persons);
+    }
+
+    @Test
+    public void sortPersonsByLastNameThenFirstNameUsingArraysSortAnonymousComparator() {
+        Person[] persons = getPersons();
+
+        // TODO использовать Arrays.sort
+        Arrays.sort(persons, new Comparator<Person>() {
+            @Override
+            public int compare(Person person1, Person person2) {
+                final String firstSirname = person1.getLastName();
+                final String secondSirname = person2.getLastName();
+                final String firstName = person1.getFirstName();
+                final String secondName = person2.getFirstName();
+                return firstSirname.compareTo(secondSirname)>0 ||
+                        firstSirname.compareTo(secondSirname)==0 && firstName.compareTo(secondName)>0? 1:
+                        firstSirname.compareTo(secondSirname)<0 ||
+                                firstSirname.compareTo(secondSirname)==0 && firstName.compareTo(secondName)<0? -1: 0;
```

Comment:
It's better to make ```.compareTo(...)``` operation once and save calculated value to variable (1 calculation instead 3). 

Anyway, it's hard to read this 4 lines.
You could use something like:
```Java
int lastNamesComparing = personLeft.getLastName().compareTo(personRight.getLastName();
return lastNamesComparing != 0 ? lastNamesComparing : personLeft.getFirstName().compareTo(personRight.getFirstName());
```